### DISCUSSION TITLE
New Commenting System | Tanel Poder Consulting

### DISCUSSION THREAD

**Discussion by giscus** [tanelpoder/blog-comments#38] [Announcements]
# New Commenting System | Tanel Poder Consulting

I switched my blog&rsquo;s commenting system to use Gisqus. It&rsquo;s still backed by Github (Discussions) as &ldquo;storage&rdquo;, but it doesn&rsquo;t request wide Github permissions from commenters (like Utterances used to do).
I didn&rsquo;t migrate old comments over, but they&rsquo;re still available here:
https://github.com/tanelpoder/blog-comments/issues/ For current/latest comments, just scroll to the bottom of the blog entries (or browse the corresponding GitHub Discussions page):
https://github.com/tanelpoder/blog-comments/discussions/  - Linux, Oracle, SQL performance tuning and troubleshooting - consulting & training.

https://tanelpoder.com/posts/new-commenting-system/

**Comment by tanelpoder** [comment]
this `is` a test!

**Comment by thatjeffsmith** [comment]
16 clicks later, it appears to work! 

🍺

**Reply by tanelpoder** [reply]
Ha thank you for your effort! :-)

Was it actually 16 clicks (or just a bunch of clicks)? For people who are already using GitHub regularly, should be relatively easy - especially after the 1st interaction, once logged in here with GitHub?
